src/moveit_commander/interpreter.py

```diff
@@ -557,16 +557,26 @@ def get_help(self):
         res.append("  go <dir> <dx>|      plan and execute a motion in direction up|down|left|right|forward|backward for distance <dx>")
         res.append("  go rand             plan and execute a motion to a random state")
         res.append("  plan <name>         plan a motion to the state <name>")
+        res.append("  plan rand           plan a motion to a random state")
+        res.append("  planner <name>      use planner <name> to plan next motion")
         res.append("  execute             execute a previously computed motion plan")
+        res.append("  stop                stop the active group")
```

Comment:
Looking at the source (probably around [here](https://github.com/kunal15595/ros/blob/681a8810e7264d153a9e4950873ec7afe94767f0/moveit/src/moveit_ros/planning_interface/move_group_interface/src/move_group.cpp#L694)?), this line may be incorrect and [this line](https://github.com/ros-planning/moveit_commander/pull/42/commits/a3df575bee1d1d22c204203d68ae56fc818ddffb#diff-3fad63fac0e8c61de84350f6122f8bceR566) is correct (i.e. no stopping feature for active group)?